fk/secretsend_test.go

```diff
@@ -128,27 +116,11 @@ func TestGetSecretFromFile(t *testing.T) {
 			readFileBytes: []byte(""),
 		}
 
-		_, err := getSecretFromFile("/fake/filename", fileReader, nil)
+		_, err := getSecretFromFile("/fake/filename", fileReader)
 		expectedErr := fmt.Errorf("/fake/filename is empty")
 		assert.Equal(t, expectedErr, err)
 	})
 
-	t.Run("returns error if user answers no", func(t *testing.T) {
```

Comment:
'tis a shame to delete this test and not add it back in at a higher level: this particular test was introduced because at one point we hit a nil pointer error if the user said no. It was only by luck that we found that error while messing around on the CLI.